.circleci/config.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: perl:5.26-threaded # debbuild needs threads
```

Comment:
Why not use something like `registry.access.redhat.com/ubi8/ubi`, `registry.fedoraproject.org/fedora:30`, or `debian:10`? We expect to be running from system Perl and such...

Reply:
That makes sense. Has this project defined which OSes/versions it supports so we can set a baseline Perl version?

It may also make sense to run tests on both a debian-alike and a redhat-alike.

Reply:
At least on the openSUSE Build Service, it's supported on Debian 7 and higher ([`Debian:debbuild`](https://build.opensuse.org/project/show/Debian:debbuild)) as well as Ubuntu 12.04 and higher ([`Ubuntu:debbuild`](https://build.opensuse.org/project/show/Ubuntu:debbuild)).

Within Fedora ([`rpms/debbuild`](https://src.fedoraproject.org/rpms/debbuild/branches)), it's been packaged for many Fedora releases, and currently is also available in EPEL for EL6 and EL7. It'll probably become available for EL8 soon enough, too.

I believe CentOS 6 is the oldest at Perl 5.10...

Reply:
Ok, using CentOS 7, for reasons explained in comments.